### PR TITLE
aragorn_out_to_gff3.py: Change Name to trnX-AAA

### PR DESCRIPTION
**Pull request by sjackman**
Add product=tRNA-Met

This naming scheme is used by Refseq. For example:
Picea abies chloroplast complete genome
[NC_021456.1](http://www.ncbi.nlm.nih.gov/nuccore/523420621)

### CODE REVIEW

**Comment by sjackman** [comment]
Example output:
```
1	aragorn	gene	361885	361960	.	-	.	ID=gene1;Name=trnM-CAT;product=tRNA-Met
1	aragorn	tRNA	361885	361960	.	-	.	ID=tRNA1;Parent=gene1;Name=trnM-CAT;product=tRNA-Met
1	aragorn	exon	361885	361960	.	-	.	Parent=tRNA1
```

**Comment by erasche** [comment]
You'll need to update the test data as well, other than that it looka good.

You can run the tests themselves with the planemo tool. http://planemo.readthedocs.org/en/latest/

**Comment by bgruening** [comment]
@sjackman this looks great! Do you want to update the test data as @erasche mentioned or should I do this?

Thanks!
Bjoern

**Comment by sjackman** [comment]
@bgruening If you have the time to update the test data, I'd appreciate it. I noticed one small flaw in this patch. It doesn't handle arbitrary length anticodons (for example, four nucleotides), which can happen with tRNA pseudogenes identified by Aragorn. I'll fix this small flaw before you merge.

File: tools/rna_tools/trna_prediction/aragorn_out_to_gff3.py [hunk 1]
```diff
@@ -10,6 +10,31 @@
...
+    'Met' : 'M',
+    'Phe' : 'F',
+    'Pro' : 'P',
+    'Ser' : 'S',
+    'Thr' : 'T',
+    'Trp' : 'W',
+    'Tyr' : 'Y',
+    'Val' : 'V',
+    'Pyl' : 'O',
+    'seC' : 'U',
```
Comment (nsoranzo) [inline tools/rna_tools/trna_prediction/aragorn_out_to_gff3.py]: Should this be 'Sec'?

Comment (sjackman) [inline tools/rna_tools/trna_prediction/aragorn_out_to_gff3.py]: No. Aragorn abbreviates selenocysteine as `seC`, though that seems to be unique to Aragorn. Wikipedia uses `Sec`.
Example Aragorn output:
`5   tRNA-seC               [244644,244785]	87  	(tca)i(29,57)`

**Comment by sjackman** [comment]
> It doesn't handle arbitrary length anticodons (for example, four nucleotides), which can happen with tRNA pseudogenes identified by Aragorn.

Fixed and ready to merge. Thanks for your offer to update the tests, @bgruening

**Comment by sjackman** [comment]
Ack. One last thing. I have to change "T" to "U"

**Comment by sjackman** [comment]
Okay. Done and ready to merge.

**Comment by bgruening** [comment]
Fixed tests and added the --full option back: https://github.com/bgruening/galaxytools/commit/74fbccd4f8b573696f3a611164132bbad346c15c

Thanks Shaun!

**Comment by sjackman** [comment]
Thanks, Björn!

**Pull request merged**